src/features/user/TenantDashboard/TenantDashboard.module.scss

```
@@ -0,0 +1,216 @@
+  width: 100%;
+}
+
+.reportControls {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 24px;
+  flex-wrap: wrap-reverse;
+}
+// Date range styles
```

Stylelint (17.3.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)